README.md

```diff
@@ -1,85 +1,50 @@
-  * SpentBy link added to the voted tickets display.
-- 0.0.1 - Initial release for mainnet operations
-
-## v1.1.0 Architecture
-
-![Stake Pool Architecture](https://i.imgur.com/2JDA9dl.png)
-
-- It is highly recommended to use 3 dcrd+dcrwallet+stakepoold nodes for
-  production use on mainnet.
-- The architecture is subject to change more in the future to lessen the
-  dependence on dcrwallet and MySQL.
-
-## v1.1.0 Migration Guide (Work-In-Progress/Subject To Change)
-
+dcrstakepool is a web application which coordinates generating 1-of-2 multisig
+addresses on a pool of [dcrwallet](https://github.com/decred/dcrwallet) servers
+so users can purchase [proof-of-stake tickets](https://docs.decred.org/mining/proof-of-stake/)
+on the [Decred](https://decred.org/) network and have the pool of wallet servers
+vote on their behalf when the ticket is selected.
+
+## 1.1.0 Release notes
+  * Per-ticket votebits were removed in favor of per-user voting preferences.
+    A voting page was added and the API upgraded to v2 to support getting and
+    setting user voting preferences.
+  * Addresses from the wallet servers which are needed for generating the 1-of-2
+    multisig ticket address are now derived from the new votingwalletextpub
+    config option. This removes the need to call getnewaddress on each wallet.
+  * An experimental daemon (stakepoold) that votes according to user preference
+    is available for testing on testnet. This daemon is not for use on mainnet
+    at this time.
+
+## 1.1.0 Mainnet Upgrade Guide
+
+- Upgrade to dcrwallet 1.0.0 or later. The deprecated enablestakemining option
+  was removed.  Replace enablestakemining with enablevoting if it is still
+  present in dcrwallet.conf.
 - Set votingwalletextpub in dcrstakepool.conf by getting the masterpubkey from
   the default account on a voting wallet per the instructions below.
-- Start dcrstakepool without any stakepoold options set. VoteBits and VoteBitsVersion
-  columns will be added to MySQL.
-- Ensure that dcrstakepool is functioning normally.
+- [Update](#updating) dcrstakepool and its dependencies.
+- Start dcrstakepool. Verify that VoteBits and VoteBitsVersion
+  columns were added to MySQL.  You should see the VoteBitsVersion being
+  adjusted for all users to the stake version that dcrwallet is configured to
+  use for voting (currently 4 on mainnet).  Also verify that the voting page
+  shows the 2 mainnet agendas that are up for vote.
+
+## 1.1.0 Testnet/stakepoold Guide (Work-In-Progress/Subject To Change)
+
+- Follow the mainnet directions. The stake version on testnet is 5 and there
+  is 1 agenda.
 - Enable MySQL access from the stakepoold hosts.
 - Configure stakepoold by editing stakepoold.conf.  You should place stakepoold
-  on the same server as dcrd/dcrwallet so stakepoold can talk to them via loopback
-  to lower latency as much as possible.
+  on the same server as dcrd/dcrwallet so stakepoold can talk to them via
```

Comment:
to clarify my comment, I assume it goes at `~/.stakepoold/stakepoold.conf`, but it would be helpful to make that explicit